docs/getting-started/launching-one-node/docker.md

```diff
@@ -18,23 +18,41 @@ To build a docker image, the following are the prerequisites which should be ins
 
 ### Build
 
-To build the docer image, the following steps need to be executed:
+To build the docker image, the following steps need to be executed:
 
   1. To clone the repository, first, create required directories using the command `mkdir -p github.com/hyperledger-labs`
   2. Change the current working directory to the above created folder by issing the command `cd github.com/hyperledger-labs`
   3. Clone this repository with `git clone https://github.com/hyperledger-labs/orion-server`
   4. Change the current working directory to the repository root directory by issuing `cd orion-server`
-  5. To build the docker images, issue the command `make docker` which will create a docker image named `orion-server`
+  5. Optional. Generate all cryptographic materials. For more info see [crypto-materials](crypto-materials).
+  6. To build the docker images, issue the command `make docker` which will create a docker image named `orion-server`
 
 ### Start Orion in a Docker Container
-Before starting the container, we need to create cryptographic materials needed to run the Orion.  To create a minimal set of cryptographic materials, execute the `cryptoGen.sh` script as shown below.
+If you plan only try Orion, you can use pre-existing configuration and crypto materials stored in folder `deployment`.
+
+To do that, you just have to run following:
+```
+docker run -it --rm -v $(pwd)/deployment/crypto/:/etc/orion-server/crypto -p 6001:6001 -p 7050:7050 orion-server
+``` 
+This command will use store all database data inside running container, so container termination will remove all data.
+
+If you plan more that just trying Orion, first you have to create __new__ and __unique__ cryptographic materials needed to run the Orion and optional, update Orion server configuration.   
+
+To create a minimal set of cryptographic materials, execute the `cryptoGen.sh` script as shown below.
 ```
-./scripts/cryptoGen.sh sampleconfig
+./scripts/cryptoGen.sh deployment
 ```
+It will create new set of crypto materials, including Certificate Authority certificate/key pair, _server_ certificate and key and certificate/key pairs for _admin_ and _user_ database users.
+
+#### Existing volumes
+As mentioned before, by default, ledger+database data and server configuration stored inside docker container, but can be overwritten by mapping host folders to container volumes.
 
-Please note that `samplconfig` folder already contains multiple configuration files used to build Orion docker images.
+Newly created docker image has three mappable volumes:
+* `/etc/orion-server/config` - contains server configuration `yml` files. Mapping is optional, only if you want to use your own configuration. Sample configuration files are stored in `deployment/config-docker` folder.
+* `/var/orion-server/ledger` - blockchain ledger storage folder. Mapping is optional, but using without mapping termination of docker container will erase all ledger data...
```

Comment:
but without mapping termination of the docker container will erase all ledger data.